### PR TITLE
265656-When did you export the waste

### PR DESCRIPTION
**Pull request by AtosSK**

### CODE REVIEW

File: EPRN.Common.Data/Migrations/20240117153148_Add-Category-Field.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    public partial class AddCategoryField : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "Category",
+                table: "WasteJourney",
+                type: "int",
+                nullable: true);
```
Comment (NikPeters-Atos) [inline EPRN.Common.Data/Migrations/20240117153148_Add-Category-Field.cs]: I think the category is determined at waste record creation. Therefore this should not be nullable. Please check if this is the case, and change if I am correct

File: EPRN.Waste.API/Repositories/Interfaces/IRepository.cs [hunk 2]
```diff
@@ -48,5 +50,7 @@ public interface IRepository
         Task<bool> Exists(int journeyId);
 
         Task<string> GetWasteNote(int journeyId);
+
+        Task<Category?> GetCategory(int journeyId);
```
Comment (NikPeters-Atos) [inline EPRN.Waste.API/Repositories/Interfaces/IRepository.cs]: I believe all waste journeys have the category set as soon as they are created, therefore Category does not need to be nullable. Please check and change if i am correct

File: EPRN.Common.Data/Migrations/20240117153148_Add-Category-Field.cs (hunk 1 above)
Comment (AtosSK) [reply EPRN.Common.Data/Migrations/20240117153148_Add-Category-Field.cs]: Done.

File: EPRN.Waste.API/Repositories/Interfaces/IRepository.cs (hunk 2 above)
Comment (AtosSK) [reply EPRN.Waste.API/Repositories/Interfaces/IRepository.cs]: Done.

**Review by NikPeters-Atos — APPROVED** [review]

**Pull request merged**